alvr/client_openxr/src/lobby.rs

```diff
@@ -208,6 +209,7 @@ impl Lobby {
                             .image_rect(rect),
                     ),
             ],
+            true,
```

Comment:
Note to self: Need to change this when adding back some form of toggling

Reply:
Technically you don't but might be good to set to false if you know the layer is opaque. The runtime might do some optimizations